packages/core/apps/core-apps-server-internal/BUILD.bazel

```diff
@@ -0,0 +1,142 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "core-apps-server-internal"
+PKG_REQUIRE_NAME = "@kbn/core-apps-server-internal"
+
+SOURCE_FILES = glob(
+  [
+    "**/*.ts",
+    "src/**/*.css",
```

Comment:
On a whim, I took the same approach as previously when handling code that includes stylesheets.